README.md

```diff
@@ -82,29 +82,33 @@ to run a simple example with Alluxio.
 
 To report bugs, suggest improvements, or create new feature requests, please open a [Github Issue](https://github.com/alluxio/alluxio/issues). Our previous [Alluxio JIRA system](https://alluxio.atlassian.net) has been deprecated since December 2018.
 
-## Depend on Alluxio
-
-For Alluxio versions 1.4 or earlier, use the `alluxio-core-client` artifact.
-
-For Alluxio versions 1.5 or later, Alluxio provides several different client artifacts. The Alluxio
-file system interface provided by the `alluxio-core-client-fs` artifact is recommended for the best
-performance and access to Alluxio-specific functionality. If you want to use other interfaces,
-include the appropriate client artifact. For example, `alluxio-core-client-hdfs` provides a client
-implementing HDFS's file system API.
-
-### Apache Maven
-```xml
-<dependency>
-  <groupId>org.alluxio</groupId>
-  <artifactId>alluxio-core-client-fs</artifactId>
-  <version>2.0.1</version>
-</dependency>
-```
+## Depend on Alluxio Artifact
 
-### SBT
-```
-libraryDependencies += "org.alluxio" % "alluxio-core-client-fs" % "2.0.1"
-```
+Alluxio publishes several client artifacts. Include the appropriate artifact in your project.
+
++ Artifact `alluxio-core-client-fs` provides a client for [Alluxio file system API](https://docs.alluxio.io/os/user/stable/en/api/FS-API.html#alluxio-java-api).  This is recommended for the best performance and access to all Alluxio functionality in a Java project.
++ Artifact `alluxio-core-client-hdfs` provides a client for [HDFS-Compatible file system API](https://docs.alluxio.io/os/user/stable/en/api/FS-API.html#hadoop-compatible-java-client).
++ Artifact `alluxio-shaded-client`, available in Alluxio 2.0.1 or later, includes both above
+  clients and all their dependencies but in a shaded form to prevent conflicts. This client jar is
+  self-contained but its size is also much bigger than the other two.
```

Comment:
Could you add what this client is recommended to be used for?

Reply:
done